libavfilter/x86/vf_gblur.asm

```diff
@@ -20,17 +20,371 @@
 
 %include "libavutil/x86/x86util.asm"
 
+SECTION .data
+
+gblur_vindex_width: dd 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15
+gblur_transpose_16x16_mask: dw 0xcc, 0x33, 0xaa, 0x55, 0xaaaa, 0x5555
+gblur_transpose_16x16_indices1: dq 2, 3, 0, 1, 6, 7, 4, 5
+gblur_transpose_16x16_indices2: dq 1, 0, 3, 2, 5, 4, 7, 6
+gblur_transpose_16x16_indices3: dd 1, 0, 3, 2, 5, 4, 7, 6, 9, 8, 11, 10, 13, 12, 15, 14
+
 SECTION .text
 
 ; void ff_horiz_slice_sse4(float *ptr, int width, int height, int steps,
 ;                          float nu, float bscale)
 
 %macro HORIZ_SLICE 0
-%if UNIX64
+%if cpuflag(avx512)
+cglobal horiz_slice_asm, 5, 12, 32, buffer, width, height, steps, localbuf, x, y, step, stride, remain, ptr
+%elif UNIX64
 cglobal horiz_slice, 4, 9, 9, ptr, width, height, steps, x, y, step, stride, remain
 %else
-cglobal horiz_slice, 4, 9, 9, ptr, width, height, steps, nu, bscale, x, y, step, stride, remain
+cglobal horiz_slice, 4, 9, 9 ptr, width, height, steps, nu, bscale, x, y, step, stride, remain
```

Comment:
Is this change needed ? it seems you should not remove ',' here.

Reply:
No, it's not necessary at all. It was supposed to be deleted by me accidentally.